README.Rmd

```diff
@@ -88,7 +88,9 @@ hardcoded_descriptions <- tribble(
   "sale_day_of_year", "Numeric encoding of day of year (1 - 365)",
   "sale_day_of_month", "Numeric encoding of day of month (1 - 31)",
   "sale_day_of_week", "Numeric encoding of day of week (1 - 7)",
-  "sale_post_covid", "Indicator for whether sale occurred after COVID-19 was widely publicized (around March 15, 2020)"
+  "sale_post_covid", "Indicator for whether sale occurred after COVID-19 was widely publicized (around March 15, 2020)",
+  "pin10_bldg_roll_mean", "Time-weighted five-year rolling average sale price for all units in the PIN10 (excluding the target sale)",
```

Comment:
Correct me if I'm wrong here, but I think it's technically more accurate to say that the average is for sales in the _PIN10_ rather than sales in the _building_, even though we've been informally using the term "building" to describe these features.